### DISCUSSION TITLE
0912. 排序数组

### DISCUSSION THREAD

**Discussion by giscus** [itcharge/AlgoNote#92] [Comments]
# 0912. 排序数组 | 算法通关手册

[https://algo.itcharge.cn/solutions/0900-0999/sort-an-array/](https://algo.itcharge.cn/solutions/0900-0999/sort-an-array/)

**Comment by ZaychikLiu** [comment]
快排好像也超时了

**Comment by chenxu2002** [comment]
快速排序超时了

**Comment by LeeX852** [comment]
大大，快排现在的力扣也超时了

**Comment by taiyi7** [comment]
堆排序里面的heapify()和python优先队列的heapify()这个函数定义是不是有冲突；前者是父节点堆调整，后者是把整个数组建堆
